package/Dockerfile

```diff
@@ -86,6 +86,11 @@ RUN zypper --non-interactive install --no-recommends \
   && find /usr/lib/locale/* -maxdepth 1 | grep -v -E "(en_US|cs_CZ|es_ES|de_DE|C.utf8)" | xargs rm -rf \
   && find /usr/share/locale -name "*.mo" -delete
 
+# fail when there are multiple Ruby interpreters installed
+RUN if [ `rpm -q --whatprovides "ruby(abi)" | wc -l` -gt 1 ]; then \
+  rpm -q --whatprovides "ruby(abi)"; \
+  echo "Multiple Rubies detected, most likely the system Ruby version has been upgraded."; \
+  echo "Update the versions of the included Ruby gems."; exit 1; fi
```

Comment:
OK... but now this check fails regardless of the presence of the 3.1.0 -> 3.2.0 fix.

It looks like, having 3.2.0 hardcoded in the zypper call, we should also hardcode it here. Like
```sh
OURS=3.2.0
NEWEST=`rpm` -q --whatprovides "ruby(abi)" --qf "%{version}\n" | sort | tail -n1`; if [ "$NEWEST" != "$OURS"; then ...
```

Reply:
It fails because there are still two Ruby versions. It turned out to be caused by failed yast2-storage-ng (https://build.opensuse.org/package/live_build_log/YaST:Head/yast2-storage-ng/openSUSE_Factory/x86_64), it was not yet rebuilt against Ruby 3.2.

So the check is correct.

Reply:
I see. And listing the used dependencies can be done like this:

```sh
cd YaST:Head/ci-ruby-container
osc buildinfo -d Dockerfile containers_tumbleweed
```